docs/platform/compliance/compliance-specification.md

```diff
@@ -2,30 +2,32 @@
 
 The Tizen Compliance Specification is the definitive set of requirements for software and hardware to create compliant Tizen devices. This document is the basis for Tizen Compliance Tests and eventual Tizen Compliance certification:
 
-- The following specification is the official version for Tizen Mobile profile 2.4 devices:
+- Iot profile
 
-  [Tizen 2.4 Compliance Specification for Mobile Profile](media/tizen-2.4-compliance-specification-for-mobile-profile-v1.0.pdf) ![PDF icon](media/application-pdf.png)
-
-- The following specification is the official version for Tizen Wearable profile 2.3.1 devices:
+  [Tizen 6.0 Compliance Specification for Iot Profile](media/tizen-6.0-compliance-specification-for-iot-profile-v0.97-beta.pdf) ![PDF icon](media/application-pdf.png)
 
-  [Tizen 2.3.1 Compliance Specification for Wearable Profile](media/tizen-2.3.1-compliance-specification-for-wearable-profile-v1.0.pdf) ![PDF icon](media/application-pdf.png)
+- TV profile:
 
-- The following specification is the official version for Tizen TV profile 2.4 devices:
+  [Tizen 5.5 Compliance Specification for TV Profile](media/tizen-5.5-compliance-specification-for-tv-profile-v0.97-beta.pdf) ![PDF icon](media/application-pdf.png)
 
   [Tizen 2.4 Compliance Specification for TV Profile](media/tizen-2.4-compliance-specification-for-tv-profile-v1.0.pdf) ![PDF icon](media/application-pdf.png)
 
-- The following specification is the beta version for Tizen TV profile 5.5 devices:
-
-  [Tizen 5.5 Compliance Specification for TV Profile](media/tizen-5.5-compliance-specification-for-tv-profile-v0.96beta2.pdf) ![PDF icon](media/application-pdf.png)
-
-- The following specification is the official version for Tizen IVI profile 3.0 devices:
+- IVI profile:
 
   [Tizen 3.0 Compliance Specification for IVI Profile](media/tizen-3.0-compliance-specification-for-ivi-profile-v1.0_1.pdf) ![PDF icon](media/application-pdf.png)
 
   [Tizen 3.0 Crosswalk Web Runtime API for IVI Profile](media/tizen-3.0-crosswalk-apis-for_ivi-v1.0_0.pdf) ![PDF icon](media/application-pdf.png)
 
-- The following specification is the official version for Tizen 3.0 devices:
+- Tizen 3.0 devices:
 
   [Tizen 3.0 Web Runtime Core Specification](media/tizen-3.0-wrt-core-spec.pdf) ![PDF icon](media/application-pdf.png)
 
+- Mobile profile
```

Comment:
To maintain consistency, please add colon:

- Mobile profile:

Reply:
Removed all ":" for consistency